.github/workflows/lintrunner_ci.yml

```diff
@@ -110,17 +110,13 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
-    if: "startsWith(github.ref, 'refs/tags/lintrunner/v*')"
```

Comment:
I'm curious on why this line is removed, for testing?

Reply:
Oh yess! Thanks for the catch